docs/docs/technology/awards.md

```diff
@@ -1,33 +1,23 @@
-
-[^emnlp2024-award]: sign.mt. 2024. [Twitter Post](https://x.com/signmt_/status/1857181686045540787).
-
-- Dr. Amit Moryossef (among others) received the `SwissNLP Award` for his "outstanding contribution to Sign Language Translation in Switzerland"[^swissnlp2024-award].
+Timothy Comment: Not ideal for reader/user, needs to read the award in the top, then scroll down (or click the number for source) then click the source down below again. One click to much. 
 
-[^swissnlp2024-award]: SwissNLP. 2024. [SwissNLP Award](https://swissnlp.org/home/activities/swissnlp-award/).
+## 2024
 
-- sign.mt was independently reviewed for Kazakh-Russian Sign Language translation, reporting accuracy of 37% compared to 98% for human translation[^kazakh-russian-review].
+- We received an `EMNLP Outstanding Demo Paper Award` for our demonstration of "sign.mt: Real-Time Multilingual Sign Language Translation Application". sign.mt. 2024. [Twitter Post](https://x.com/signmt_/status/1857181686045540787).
 
-[^kazakh-russian-review]: Imashev et al. 2024. [Comparative Analysis of Sign Language Interpreting Agents Perception: A Study of the Deaf](hhttps://aclanthology.org/2024.lrec-main.319/).
+- Dr. Amit Moryossef (among others) received the `SwissNLP Award` for his "outstanding contribution to Sign Language Translation in Switzerland". SwissNLP. 2024. [SwissNLP Award](https://swissnlp.org/home/activities/swissnlp-award/).
 
-- Using our technology (code, models, and app), a team from India won[^sih] the [Smart India Hackathon](https://www.sih.gov.in/).
+- sign.mt was independently reviewed for Kazakh-Russian Sign Language translation, reporting accuracy of 37% compared to 98% for human translation. Imashev et al. 2024. [Comparative Analysis of Sign Language Interpreting Agents Perception: A Study of the Deaf](hhttps://aclanthology.org/2024.lrec-main.319/).
 
-[^sih]: Prasad et al. 2024. [LinkedIn Post](https://www.linkedin.com/posts/suraj-prasad-b2290b24b_smartindiahackathon2024-indiansignlanguage-activity-7275070354627194880-fWVF/).
+- Using our technology (code, models, and app), a team from India won the [Smart India Hackathon](https://www.sih.gov.in/). Prasad et al. 2024. [LinkedIn Post](https://www.linkedin.com/posts/suraj-prasad-b2290b24b_smartindiahackathon2024-indiansignlanguage-activity-7275070354627194880-fWVF/).
 
 ## 2023
 
-- Dr. Amit Moryossef (among others) received the `ACL Outstanding Paper Award` for "Considerations for Meaningful Sign Language Machine Translation"[^acl2023-award].
-
-[^acl2023-award]: Müller et al. 2023. [Considerations for Meaningful Sign Language Machine Translation](https://aclanthology.org/2023.acl-short.60/).
+- Dr. Amit Moryossef (among others) received the `ACL Outstanding Paper Award` for "Considerations for Meaningful Sign Language Machine Translation". Müller et al. 2023. [Considerations for Meaningful Sign Language Machine Translation](https://aclanthology.org/2023.acl-short.60/).
```

Comment:
One of the reasons for having the citation as the link, is that it would remove the redundancy of writing the name of the paper twice, here and elsewhere